Core/config.h

```diff
@@ -1,7 +1,7 @@
 #ifndef _CONFIG_H_
 #define _CONFIG_H_
 
-#include "license.txt" 
```

Comment:
all good changes but not related to linux usb. can we do these changes in a separate change?

Reply:
Reverted all except QUEUE_SIZE change